config/config-feature-flags.yaml

```
@@ -108,7 +108,7 @@ data:
   # Setting this flag will determine how Tekton pipelines will handle extracting results from the task.
   # Acceptable values are "termination-message" or "sidecar-logs".
   # "sidecar-logs" is an experimental feature and thus should still be considered
```

with this promotion to beta, the feature is no longer experimental, please update 

```suggestion
  # "sidecar-logs" is
```

---

Fixed!